frontend/src/components/pages/connect/Overview.tsx

```diff
@@ -95,28 +110,28 @@ class TabClusters extends Component {
                             }
 
                             return (
-                                <span className="hoverLink" style={{display: 'inline-block', width: '100%'}} onClick={() => appGlobal.history.push(`/connect-clusters/${encodeURIComponent(r.clusterName)}`)}>
-                                        {r.clusterName}
-                                    </span>
+                                <span className="hoverLink" style={{ display: 'inline-block', width: '100%' }} onClick={() => appGlobal.history.push(`/connect-clusters/${encodeURIComponent(r.clusterName)}`)}>
```

Comment:
nit: should we use components from the UI package for the links instead of custom classes? In case there is something missing, I guess we can make a request for the package?